includes/class-wc-taxjar-integration.php

```diff
@@ -1047,12 +1068,38 @@ public function calculate_renewal_order_totals( $order, $subscription, $type ) {
 			return $order;
 		}
 
+		/**
+		 * Determines whether or not TaxJar should calculate tax on an order
+		 * @param $order
+		 * @return bool
+		 */
+		public function should_calculate_order_tax( $order ) {
+			$should_calculate = true;
+
+			// Cannot use $order->get_total() as it is not available yet when creating orders through the REST API
+			$total = 0;
+			foreach( $order->get_items( 'line_item', 'shipping', 'fee' ) as $item ) {
+				$total += $item->get_total();
+			}
+
+			if ( $total === 0 ) {
+				$should_calculate = false;
+			}
+
+			return apply_filters( 'taxjar_should_calculate_order_tax', $should_calculate );
```

Comment:
What do you think about adding `$order` as an additional filter argument?

```
return apply_filters( 'taxjar_should_calculate_cart_tax', $should_calculate, $order );
```

That would allow someone to conditionally calculate tax based order.

Reply:
Same here as with the cart filter, definitely a change we will make before release. Thanks!